uber/site_sections/summary.py

```diff
@@ -225,13 +225,13 @@ def restricted_untaken(self, session):
                     untaken[job.location][hour].append(job)
         flagged = []
         for attendee in attendees:
-            if attendee.trusted and not attendee.is_dept_head:
+            if not attendee.is_dept_head:
                 overlapping = defaultdict(set)
                 for shift in attendee.shifts:
                     if not shift.job.restricted:
                         for dept in attendee.assigned_depts_ints:
                             for hour in shift.job.hours:
-                                if hour in untaken[dept]:
+                                if attendee.trusted_in(dept) and hour in untaken[dept]:
                                     overlapping[shift.job].update(untaken[dept][hour])
```

Comment:
alright I'm going to slouch on the unit tests for just this one report, though, I tested it by hand and it works.  I added a ton of new tests to this code already